certified-connectors/Tilkee-V3/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2241 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.3",
+    "description": "Tilkee is an online platform that makes cross-functional collaboration happen. Combining deep knowledge of sales enablement with augmented intelligence, the vision of the team behind Tilkee is to align marketing with sales and guide Sales, Product Marketing, Channel Enablement, and Customer Success teams towards digital transformation.",
+    "title": "Tilkee V3",
```

Comment:
@FloxoS Please revert back the title as it is not allowed to change the title and with version numbers.